rest/admin_api.go

```diff
@@ -62,6 +62,9 @@ func (h *handler) handleCreateDB() error {
 
 		persistedConfig.cas, err = h.server.bootstrapContext.connection.InsertConfig(bucket, h.server.config.Bootstrap.ConfigGroupID, persistedConfig)
 		if err != nil {
+			if errors.Cause(err) == base.ErrAuthError {
+				return base.HTTPErrorf(http.StatusForbidden, "auth failure accessing provided bucket %s", bucket)
```

Comment:
Should we indicate here that the access failure occurs using the bootstrap credentials (as opposed to the credentials used to auth the admin API REST call)?  

I'm also curious whether 'auth' was left intentionally vague - if this is authorization (and not authentication), I think using authorization failure is a better choice.

Reply:
Can add info about bootstrap creds.
Yes auth was left vague. As with the above comment, we don't really know which it is.